docs/book/src/topics/custom-images.md

```diff
@@ -63,9 +63,9 @@ See [Upgrading workload clusters][upgrading-workload-clusters] for more details.
 
 To use a custom image, it needs to be referenced in an `image:` section of your `AzureMachineTemplate`. See below for more specific examples.
 
-### Using Shared Image Gallery (Recommended)
+### Using Compute Gallery (Recommended)
 
-To use an image from the [Shared Image Gallery][shared-image-gallery], fill in the `resourceGroup`, `name`, `subscriptionID`, `gallery`, and `version` fields:
+To use an image from the [Compute Gallery][compute-gallery], previously known as Shared Image Gallery (SIG), fill in the `resourceGroup`, `name`, `subscriptionID`, `gallery`, and `version` fields:
```

Comment:
the official name has "Azure" in it : Azure Compute Gallery

Reply:
I feel like `Azure` might be a redundant word here, given that it's the Azure provider in general.

Reply:
either is probably fine here, `Azure Compute Gallery` is probably more technically correct

Reply:
+1 to using the official product name in docs, "Azure Compute Gallery" even if it's redundant

Reply:
Done. We now consistently use either `Azure Compute Gallery` or `private compute gallery`, which seem correct.